umh-core/pkg/constants/dataflowcomponent.go

```diff
@@ -22,3 +22,15 @@ const (
 	// So by setting this to 50 ms, we can ensure that an instance will never start if it triggers the alerting threshold
 	DataflowComponentExpectedMaxP95ExecutionTimePerInstance = time.Millisecond * 50 // needs to be higher than BenthosExpectedMaxP95ExecutionTimePerInstance
 )
+
+const (
+	// Used to set the context timeout for updating the observed state of a DataflowComponent instance
+	DataflowComponentUpdateObservedStateTimeout = time.Millisecond * 5
+)
+
+const (
+	// WaitTimeBeforeMarkingStartFailed is the time before marking a DataflowComponent instance as Startfailed if the underlying benthos has not started and stable
+	// Benthos takes some time to start usually and we give an enough buffer time of 15 seconds
+	// Default value is 15 seconds
+	WaitTimeBeforeMarkingStartFailed = time.Second * 15
```

Comment:
15 secs seems really high.

Reply:
Benthos has a wait time in there of I think 10 seconds, so it waits for 10 seconds until the healthchecks have passed, therefore this also needs to wait this long